controller/controller.go

```diff
@@ -160,6 +187,12 @@ func (c controller) Submit(req Request) (*task.Task, error) {
 			}
 		}
 
+		closer := make(chan struct{})
+		err = c.WaitForStatus(req, StatusStopped, closer)
```

Comment:
This is already supported. I favored the named argument so a human reading this code at least gets an idea of what `nil` actually is.